_ingest-pipelines/processors/append.md

```diff
@@ -43,6 +43,7 @@ Parameter | Required/Optional | Description |
 `description`  | Optional  | A brief description of the processor.  |
 `if` | Optional | A condition for running the processor. |
 `ignore_failure` | Optional | Specifies whether the processor continues execution even if it encounters errors. If set to `true`, failures are ignored. Default is `false`. |
+`allow_duplicates` | Optional | Specifies whether to append the values that the field already contains. If `true`, duplicate values are appended, otherwise they are skipped. |
```

Comment:
```suggestion
`allow_duplicates` | Optional | Specifies whether to append the values already contained in the field. If `true`, duplicate values are appended. Otherwise, they are skipped. |
```